docs/batch-changes/create-a-batch-change.mdx

```diff
@@ -180,6 +180,14 @@ You can view a list by clicking the **Batch Changes** icon in the top navigation
 
 ![batch-change-icon-to-view](https://sourcegraphstatic.com/docs/images/batch_changes/2024/batch_changes_icon_in_menu.png)
 
+### Title-based search
+
+You can search through your previously created batch changes by title. This search experience makes it much easier to find the one you’re looking for, especially when you have large volumes of batch changes to monitor.
+
+Start typing the keywords that match the batch change’s name, and you will see a list of relevant results.
+
+![title-based-search-batch-changes](https://storage.googleapis.com/sourcegraph-assets/Docs/title-based-search-bc.png)
```

Comment:
I tried pulling down this branch in order to see this pic in my IDE (hoping it'd render), but I couldn't :( I'm assuming it's right!